lib/fizzy/stack.hpp

```diff
@@ -0,0 +1,31 @@
+#pragma once
+#include <cstdint>
+#include <vector>
+
+namespace fizzy
+{
+template <typename T>
+class stack : public std::vector<T>
+{
+public:
+    using difference_type = typename std::vector<T>::difference_type;
+
+    using std::vector<T>::vector;
+
+    using std::vector<T>::back;
```

Comment:
Are these required? (I think they are not with public inheritance?)

And with these here can we use private inheritance?

Reply:
I don't know why, but somehow they are required. Otherwise you cannot use vector's method from within the stack class definition.

There are many other methods used from outside so private inheritance will not work.

Reply:
I see, maye something like `std::vector<T>::back()` would have worked, too, but these might look better.

Reply:
This is strange and unexpected compilation error, looks like related to which methods depend on T type in `std::vector`. Also `vector::vector` does not work any more, you need `std::vector<T>::vector`. I didn't bother to investigate longer than 15 mins.

Reply:
Perhaps https://github.com/wasmx/fizzy/blob/master/lib/fizzy/execute.cpp#L283 replacing this with `stack.peek()` would have helped?

Reply:
The `peek()` looks weird. The `peek(1)` means the top item and it unnecessary uses `end()` instead of just using `[]` operator. I might touch it in the "labels" PR.

Reply:
Can you just create another clean up PR for stack?